src/sdk/search/Provider.tsx

```diff
@@ -71,7 +79,7 @@ export const SearchProvider: FC<Props> = ({
   children,
   pageInfo,
 }) => {
-  const [pages, setPages] = useState([initalState.page])
+  const [pages, setPages] = useState(() => [initalState.page])
```

Comment:
Can we avoid having two functions with names `setPage` and `setPages`? They're being used in close proximity and inside the callback for `useMemo` they're being used in the same scope.

How about calling the innermost one (inside the callback) `setCurrentPage`, or something like that? Just to make them easier to differentiate and improve readability.

Reply:
what about setPage -> pushPage?

Reply:
Yep, sounds good 😄